doc/sphinx/source/getting_started/inputdata.rst

```diff
@@ -47,8 +80,6 @@ A list of the datasets for which a cmorizers is available is provided in the fol
 +------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
 | CRU                          | tas, pr (Amon)                                                                                       |   2  | Python          |
 +------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
-| Duveiller2018                | albDiffiTr13                                                                                         |   2  | Python          |
```

Comment:
Why is this removed?

Reply:
Because there is now a link to the source file in the github repo. This avoids having to synchronize the docs whenever the example recipe is updated. That's how I interpreted @bouweandela 's comment (https://github.com/ESMValGroup/ESMValTool/pull/1330/files/b6de044f6f982b71149b1476cd2dd17d477c1127#r328653971). Do you prefer to keep it?

Reply:
But why removing only this particular dataset? This table is just an overview of all available cmorizers. Either we remove it completely or we keep it.